src/test/osd/osd-scrub-repair.sh

```diff
@@ -17,15 +17,22 @@
 source $(dirname $0)/../detect-build-env-vars.sh
 source $CEPH_ROOT/qa/workunits/ceph-helpers.sh
 
+if [ `uname` = FreeBSD ]; then
```

Comment:
@tchaikov @dzafman 
The diff is only there for debugging and diagnostic reasons there.
So my understanding would be that the output would be worthwhile to examine